client/src/components/Markdown/Elements/InvocationTime.vue

```diff
@@ -19,7 +19,8 @@ export default {
     computed: {
         content() {
             const invocation = this.invocations[this.args.invocation_id];
-            return invocation && invocation["create_time"];
+            const iso = new Date(invocation && invocation["create_time"]);
+            return iso.toUTCString();
```

Comment:
```suggestion
            return invocation && new Date(invocation["create_time"])).toUTCString();
```
otherwise I think this fails if invocation is null